src/app.js

```diff
@@ -76,36 +74,68 @@ class App extends Component {
     });
   }
 
-  // update source/target nodes from url
-  updateNodesFromUrl() {
+  // update source/target nodes, checked metapaths, etc from url
+  updateStateFromUrl(event) {
     this.props.dispatch((dispatch) => {
```

Comment:
Will look into this.

Reply:
I think I'm going to put this on hold for now, since this problem exists all throughout the app, due to my inexperience with the Redux paradigm. I've created issue #64 to address this at a later time, as well as some other issues you may have noticed (such as files being too many lines).